### DISCUSSION TITLE
Hide power states?

### DISCUSSION THREAD

**Discussion by plauk-emag** [Ylianst/MeshCentral#5196] [Q&A] [answered]
We like to get rid of our old remote management tool and MeshCentral fits all our needs. As a German company, we have a strong works council that will definitely directly criticize the display of the power state, because it could be used to draw conclusions about when the users are working. Is it possible to hide the 7-day power state? I have already tried to find information on the Internet and unfortunately could not find any settings or other information. Thanks a lot!

**Comment by si458** [comment]
i dont believe its currently possible,
but looking at the HTML code, all you have to do is set `display:none` in the css of the `p10html2` div and it gets hidden for you!
so you could submit a PR with the option in the user preferences or as a setting, and when it starts up it will hide it for you?

**Reply by plauk-emag** [reply]
Thank you, the `display:none `did the trick and the data is not shown anymore. 
Unfortunaltely I'm new to Github and not a developer so I cannot do any changes here. But if anyone would be able to help here this would be greatly appreciated!

**Reply by si458** [reply]
ill have a look when i get chance (community dev here)

**Reply by si458** [reply, marked as answer]
#5396

Also PR here to add feature in for you - #5398
